hypha/apply/funds/models/forms.py

```diff
@@ -211,9 +211,46 @@ def __str__(self):
         return self.form.name
 
 
+class AbstractRelatedProjectSOWForm(Orderable):
```

Comment:
Can you please add a docstring on why this Class is needed. IT's not very clear from it's implimentaton.

Reply:
I have added a docstring there, it is just getting copied from the above forms. Its main function is to attach the SOW forms to the funds and labs.